recipes/unbaffeld/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "unbaffeld" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  nameu: "{{ name|upper }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ nameu }}-{{ version }}.tar.gz
+  sha256: ac032334a2ca1e5f1f783b12e96610b582ad2f59bc40af2a615cd25a8c54b120
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python >=3.7
+    - inference-tools >=0.12.0
+    - numpy >=1.8
+    - scipy >=1.6.3
+    - h5py
+    - matplotlib >=3.4.2
+
+test:
+  imports:
```

Comment:
You only need to import this package, not it's dependencies

Reply:
done